TKGame/BackEnd/Music.cs

```diff
@@ -23,7 +23,7 @@ public class Music
         public static void LoadContent(ContentManager Content, float vol)
         {
             //Song is just temporary until Keegan makes us a basedAF soundtrack
-            backgroundSong = Content.Load<Song>(@"Audio/Break-Out-Loop");
+            backgroundSong = Content.Load<Song>(@"C:/Users/Greywater/Documents/Git/TKGame/TKGame/Content/bin/DesktopGL/Audio/Break-Out-Loop");
```

Comment:
no bad please no

Reply:
use Path library, see Save/Load Stage functions

Reply:
fixed it